src/styles/blocks/header.scss

```diff
@@ -0,0 +1,104 @@
+body,
+h1,
+ul,
+li,
+p,
+a {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: $font-family;
+  background-color: #f9f9f9;
+  width: 100%;
+}
+
+.header {
+  width: 100%;
+  max-width: $max-width-container;
+  height: $header-height;
+  display: flex;
+  justify-content: space-between;
+  padding-left: $padding-container;
+  padding-right: $padding-container;
+  align-items: center;
+  background-color: #fff;
+  position: fixed;
+  font-size: $font-size-base;
+  margin: 0 auto;
```

Comment:
The 'margin: 0 auto;' style is not needed here. Since you are using flexbox, you can align items using justify-content or align-items.